daemon/create.go

```diff
@@ -14,26 +14,35 @@ import (
 	"github.com/opencontainers/runc/libcontainer/label"
 )
 
+// ContainerCreate is the parameter set to ContainerCreate()
+type ContainerCreate struct {
+	Name            string
+	Config          *runconfig.Config
+	HostConfig      *runconfig.HostConfig
+	AdjustCPUShares bool
+	FromBuilder     bool
```

Comment:
Hang on, that doesn't seem right. Today (pre this PR), a CLI using -v can use a named volume, right? And since 16650, this was still possible. However, a build was blocked from using a named volume. 

But, with this PR, an API (not CLI) caller passing something in volumes to create still has to be able to create a named volume to ensure backwards API compatibility, but I still need to block it from the builder create. Right? Hence, the reason this is needed. 

Or am I missing something?